bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -61,15 +51,7 @@ export default class ActionsBar extends Component {
             close={() => {exitAudio();}}
 
           />
-
-          <Button
-            onClick={this.handleClick}
-            label={'Cam Off'}
-            color={'primary'}
-            icon={'video-off'}
-            size={'lg'}
-            circle={true}
-          />
+          <JoinVideo />
```

Comment:
I tried adding  `style={{visibility: 'hidden'}}`  to the previously existing button, but that didn't hide it.